apps/src/templates/courseOverview/CourseEditor.js

```diff
@@ -80,6 +84,19 @@ export default class CourseEditor extends Component {
           />
         </label>
         <label>
+          Verified Resources
+          <input
+            name="has_verified_resources"
+            type="checkbox"
+            defaultChecked={this.props.hasVerifiedResources}
+            style={styles.checkbox}
+          />
+          <p>
+            Check if this course has resources for verified teachers, and we
```

Comment:
Do people know what is meant by resources? The text above says locked lessons and answer keys specifically.